platform/darwin/src/MGLGeoJSONSource.h

```diff
@@ -7,48 +7,53 @@ NS_ASSUME_NONNULL_BEGIN
 @protocol MGLFeature;
 
 /**
+ Options for `MGLGeoJSONSource` objects.
+ */
+typedef NSString *MGLGeoJSONSourceOption NS_STRING_ENUM;
+
+/**
  An `NSNumber` object containing a Boolean enabling or disabling clustering.
  If the `features` property contains point features, setting this option to
  `YES` clusters the points by radius into groups. The default value is `NO`.
  */
-extern NSString * const MGLGeoJSONClusterOption;
+extern MGLGeoJSONSourceOption const MGLGeoJSONClusterOption;
```

Comment:
Good point. Looks like the Swift compiler can trim common suffixes from Objective-C enumerations and `NS_OPTION` fields but not string enumerations. We’ll have to use `NS_SWIFT_NAME`.